src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -359,6 +359,135 @@
                     ]
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/bundle/create",
```

Comment:
The PRD states that the actions (create, delete, abort (which I know we are't doing)) are nested under `/actions`.

Reply:
Yeah, I removed `/actions` because IMO that seems unnecessary. But you are right, I'll put that back in.